app/views/register/leadtrustee/individual/MatchingFailedView.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import controllers.register.leadtrustee.individual.routes._
+
+@this(
+    main_template: MainTemplate,
+    formHelper: FormWithCSRF,
+    appConfig: FrontendAppConfig
+)
+
+@(draftId: String, index: Int, numberOfFailedAttempts: Int)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("leadTrustee.individual.matching.failed.title")
+) {
+
+    @formHelper(action = MatchingFailedController.onSubmit(index, draftId), 'autoComplete -> "off") {
+
+        @components.heading_with_caption(
+            messagePrefix = "leadTrustee.individual.matching.failed",
+            captionParam = Some(numberOfFailedAttempts.toString)
+        )
+
+        <p>@messages("leadTrustee.individual.matching.failed.paragraph1")</p>
+
+        <p>@messages("leadTrustee.individual.matching.failed.paragraph2", appConfig.maxMatchingAttempts - numberOfFailedAttempts)</p>
```

Comment:
We shouldn't be calculating the number of remaining in the view, can we pass this from the controller?